### ISSUE TITLE
Some Tickers may not have Institutional investors: IndexError: list index out of range

### ISSUE THREAD

**Issue by garython** [ranaroussi/yfinance#403]
Not all tickers have institutional investor. The call to holder[1] gives an error in the get_fundamental function. 
Can you please make a change to handle the error. Put a if condition to check length of  holders before accessing holders[1] 
Or use a try .... except.

Thanks in advance.

>>> yf.Ticker('APT.AX').info
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/yfinance/ticker.py", line 138, in info
    return self.get_info()
  File "/usr/local/lib/python3.7/site-packages/yfinance/base.py", line 415, in get_info
    self._get_fundamentals(proxy)
  File "/usr/local/lib/python3.7/site-packages/yfinance/base.py", line 286, in _get_fundamentals
    self._institutional_holders = holders[1]
IndexError: list index out of range

**Comment by JavaIsPain** [comment]
"Fixed" it by changing the respective part of the code to
`
        # holders
        try:
            url = "{}/{}/holders".format(self._scrape_url, self.ticker)
            holders = _pd.read_html(url)
            self._major_holders = holders[0]
            self._institutional_holders = holders[1]
            if 'Date Reported' in self._institutional_holders:
                self._institutional_holders['Date Reported'] = _pd.to_datetime(
                    self._institutional_holders['Date Reported'])
            if '% Out' in self._institutional_holders:
                self._institutional_holders['% Out'] = self._institutional_holders[
                    '% Out'].str.replace('%', '').astype(float)/100
        except IndexError:
            print("yfinance::IndexError::Info::Could not find any institutional holders.")
`

**Comment by threatlead** [comment]
Receiving same error for Ticker=**EXPE**

```python
  File "/python3.8/site-packages/yfinance/ticker.py", line 138, in info
    return self.get_info()
  File "/python3.8/site-packages/yfinance/base.py", line 415, in get_info
    self._get_fundamentals(proxy)
  File "/python3.8/site-packages/yfinance/base.py", line 286, in _get_fundamentals
    self._institutional_holders = holders[1]
IndexError: list index out of range
```

Thank you!

**Comment by Bruno-Hanzen** [comment]
@JavaIsPain , seems to work. You might want to submit a pull request.

**Comment by JavaIsPain** [comment]
I think that's a very bad fix so I won't submit a pull request. Not everyone wants random internal error messages printed to their console out of nowhere. Fixing the problem is a way better option, but that's not what my code does.

**Comment by uebenes** [comment]
I solved that problem with following fix - works really good: 

        holders = _pd.read_html(url)
        if len(holders) > 0:
            self._major_holders = holders[0]
        else:
            self._major_holders = _pd.DataFrame()
        if len(holders) > 1:    
            self._institutional_holders = holders[1]
        else:
            self._institutional_holders = _pd.DataFrame()

**Comment by uebenes** [comment]
But it looks, it has been fixed already with commit ea429cd034a7cf22015a8400e4ca2a17d1968ef5#diff-69e424ccdc9dbd39c5d517098950e47
